### DISCUSSION TITLE
Building a Go template playground with Wasm

### DISCUSSION THREAD

**Discussion by giscus** [kujenga/website#87] [Blog Post Comments]
# Building a Go template playground with Wasm

Creating an interactive live parser to experiment with the Go
templating language, built using WebAssembly and running the Go standard library
directly in your browser.

https://ataylor.io/blog/go-templates/

**Comment by breadchris** [comment]
awesome project!

**Reply by kujenga** [reply]
Thanks! It was fun to build.
